### PR TITLE
WFP-1880 app must handle removing team from probation estate

### PR DESCRIPTION
**Pull request by carlov20**

### CODE REVIEW

File: integration_tests/integration/choose-practitioner.spec.ts [hunk 1]
```diff
@@ -325,4 +325,80 @@ context('Choose Practitioner', () => {
       choosePractitionerPage.radio('N03F02::OM2').should('not.exist')
     })
   })
+
+  it('should not show team which does not exist in probation estate but is in user preferences', () => {
```
Comment (carlov20) [inline integration_tests/integration/choose-practitioner.spec.ts:329]: I made the calls with the old team code and this is what the workload/ allocations returned

File: integration_tests/integration/choose-practitioner.spec.ts [hunk 2]
```diff
@@ -325,4 +325,80 @@ context('Choose Practitioner', () => {
...
+            },
+            email: 'j.doe@email.co.uk',
+            grade: 'PQiP',
+            workload: 19,
+            casesPastWeek: 2,
+            communityCases: 3,
+            custodyCases: 5,
+          },
+        ],
+        OLDTEAM1: [
```
Comment (sp-watson) [inline integration_tests/integration/choose-practitioner.spec.ts:379]: So there is still a practitioner associated with the old team?

**Review by sp-watson — APPROVED** [review]
Ideally we should make sure the Ui never assumes that teams definitely exist.

But I don't think an allocation demand would be associated with an "old" team anyway so this is probably OK.

File: integration_tests/integration/choose-practitioner.spec.ts (hunk 2 above)
Comment (carlov20) [reply integration_tests/integration/choose-practitioner.spec.ts:379]: yeah there was, I made the request with Wisbech staff code and there were still practitioners in there

**Pull request merged**